### ISSUE TITLE
Wrong undo behavior when remaping kj to <esc>

### ISSUE THREAD

**Issue by inoc603** [VSCodeVim/Vim#568]
### What did you do?

I remapped kj to <esc> with the following setting.json:

```json
{
    "vim.insertModeKeyBindings": [
        {
            "before": ["k", "j"],
            "after": ["<esc>"]
        }
   ]
}
```

Then:

- enter insert mode
- change some text
- press kj to exit to normal mode
- press u

### What did you expect to happen?

The changes I made are undone.

### What happened instead?

Wrong undo result and an extra `k` character is inserted. 

### Technical details:

* VSCode Version: 1.4.0
* VsCodeVim Version: 0.1.4
* OS: OSX El Captain 10.11.5

**Comment by ScenK** [comment]
+1

**Comment by inoc603** [comment]
It seems that this bug is fixed in the v0.1.7 release along with #523 . Thanks @johnfn !

Closed this one.

**Comment by johnfn** [comment]
Missed this one when closing issues earlier. Thanks! :)
